Adjust documentation after removing jaeger (#17414)
kyma-project/kyma: docs/01-overview/main-areas/service-mesh/smsh-03-istio-sidecars-in-kyma.md

## Observability

> Furthermore, Istio proxies improve tracing: Istio performs global tracing and forwards the data to a tracing backend. One can bring own tracing backend or one can deploy [Jaeger](https://github.com/kyma-project/examples/tree/main/jaeger) inside the cluster using the [Zipkin protocol](https://zipkin.io). Learn more about  in [Tracing Architecture](../../main-areas/telemetry/telemetry-03-traces.md).

Reviewer: Let's replace Zipkin by OTLP here. That's what all the defaults for Istio tracing expect now and also the preferred way with Jaeger.

> Furthermore, Istio proxies improve tracing: Istio performs global tracing and forwards the data to a tracing backend. One can bring own tracing backend or one can deploy [Jaeger](https://github.com/kyma-project/examples/tree/main/jaeger) inside the cluster using the [Otlp protocol](https://opentelemetry.io/docs/reference/specification/protocol/). Learn more about  in [Tracing Architecture](../../main-areas/telemetry/telemetry-03-traces.md).

Reviewer 2: ```suggestion
Furthermore, Istio proxies improve tracing: Istio performs global tracing and forwards the data to a tracing backend. You can bring own trace backend or deploy [Jaeger](https://github.com/kyma-project/examples/tree/main/jaeger) inside the cluster using the [OTLP protocol](https://opentelemetry.io/docs/reference/specification/protocol/). Learn more in [Tracing Architecture](../../main-areas/telemetry/telemetry-03-traces.md).
```
